### ISSUE TITLE
Impaired executive functioning

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5850]
**Preferred term label:**
Impaired executive functioning

**Synonyms**
Executive dysfunction
Weak executie functioning

**Definition (free text, please give PubMed ID)**
An impairment in the higher-level cognitive skills that are used to control and coordinate other cognitive abilities and behaviors such as organization and regulation of actions, self-control, emotional regulation, monitoring internal and external stimuli, initiating and inhibiting context-specific behavior, moral reasoning, decision-making.
Source: https://memory.ucsf.edu/symptoms/executive-functions

**Parent term (use hpo.jax.org/app)**
'Impairment in personality functioning'

This is needed for the Kids First INCLUDE supplement.

**Comment by pnrobinson** [comment]
Adding HP_0033051
def: A disturbance of executive functioning, which is broadly defined as the set of  abilities  that  allow  for  the  planning,  executing, monitoring,  and  self-correcting  of  goal-directed behavior  while  inhibiting  task-irrelevant  behavior. At least some degree of executive skill is needed to complete most cognitive  tasks,  and  deficits  in  executive  abilities  are central  to  many  clinical  conditions,  including  fronto-temporal  dementia.
https://pubmed.ncbi.nlm.nih.gov/28631987/
